flatdata-rs/lib/src/test/test_generated.rs

```diff
@@ -1,4 +1,5 @@
 // Do not edit: This code was generated by flatdata's generator.
+#[cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
I would actually prefer to have it formatted, because: 

1. This is file is a good reference for generated code.
2. The diff is smaller.

Reply:
I actually prefer it exactly the other way round: I want to see what is actually generated. But that is a separate topic, so I will remove from this review.

Reply:
Added it back, it was not formatted previously, and without this the diff is much bigger (did it manually for now, but I would prefer to rather have this in the generator itself)